jujugui/static/gui/src/app/app.js

```diff
@@ -437,7 +437,7 @@ YUI.add('juju-gui', function(Y) {
             '#user-name .header-banner__link--breadcrumb');
         var auth = this._getAuth('charmstore');
         if (breadcrumbElement) {
-          breadcrumbElement.textContent = auth && auth.user && auth.user.name ||
+          breadcrumbElement.textContent = auth && auth.user ||
```

Comment:
Yeah, I talked with @rogpeppe and the data structures for JEM and charmstore are nearly identical. Furthermore they pass through a tranformation callback that makes them completely identical on our side.